front/src/store/index.js

```diff
@@ -50,11 +60,42 @@ export default new Vuex.Store({
     },
     logout ({ commit }) {
       commit('CLEAR_USER_DATA');
+    },
+    recoveryPasswordSendEmail ({ commit }, payload) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword`,
+          { Email: payload.email }).then(resp => {
+          if (resp.status !== 200) {
+            reject(resp.data);
+          } else {
+            commit('SET_EMAIL_DATA', payload.email);
```

Comment:
why do we need this?

Reply:
what is it, a commit or a status code check?